library/Fission/Platform/User.hs

```diff
@@ -1,5 +1,5 @@
-module Fission.Platform.Heroku.UserConfig
-  ( UserConfig (..)
+module Fission.Platform.User
```

Comment:
Hmm, thoughts on getting this out of `Platform` (platforms are like: Heroku, GitHub Apps, &c)? Maybe `Fission.User.Provision.Types`. You could still do this afterwards:

```haskell
import qualified Fission.User.Provision.Types as User
```

Thoughts?

Reply:
Yeah makes a lot of sense :+1: